src/pages/dogs/index.jsx

```diff
@@ -110,10 +137,16 @@ export default function DogsPage() {
                 dogs={dogs}
             />}
 
-            {!loading && !limitedAssociation && <DogTable
-                loading={loading}
-                dogs={dogs}
-            />}
+          {!loading && !limitedAssociation ? (
+              dogs.length === 0 ? (
+                <div style={{ textAlign: 'center', fontWeight: 'bold', fontSize: '20px' }}> 
```

Comment:
Filter bars should not be shown in the `loading` state.
Filter bars should show even if no dogs are found:
![Dashboard No Dogs](https://github.com/GTBitsOfGood/canine-assistants/assets/45372449/44055474-1a7e-406a-bed3-efb8aa4e97d1)